hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/TestOmConfig.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.om;
+
+import org.apache.hadoop.hdds.conf.MutableConfigurationSource;
+import org.apache.hadoop.hdds.conf.OzoneConfiguration;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TestOmConfig {
```

Comment:
Simple Javadoc.

Reply:
I'm not sure comments like the following are very useful, but I can add it if you prefer.

```java
/**
 * Tests for {@link OmConfig}.
 */
```

Reply:
Agreed that test javadocs do not really add much info. I have no preference, just most test classes contain top level comments.

Reply:
I think that's a legacy from JUnit4, which required public test classes (and methods), and checkstyle requires javadoc on public types.

Reply:
Ok noted, thanks for the info.